pkg/webhook/pod/v1/pod_default.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2023 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the License);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an AS IS BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	nodeutil "github.com/openyurtio/openyurt/pkg/controller/util/node"
+	podutil "github.com/openyurtio/openyurt/pkg/controller/util/pod"
+	"github.com/openyurtio/openyurt/pkg/controller/yurtcoordinator/constant"
+)
+
+// Default satisfies the defaulting webhook interface.
+func (webhook *PodHandler) Default(ctx context.Context, obj runtime.Object, req admission.Request) error {
+	po, ok := obj.(*v1.Pod)
```

Comment:
I think we should only handle pod status update requests initiated by kube-controller-manager

Reply:
yes, i set the MutatingWebhookConfiguration for resource is pods/status, and the operations is update, so only the pods/status update will handle